week2-assignment/styles.css

```
@@ -0,0 +1,207 @@
+@import "reset.css";
+
+body {
+  background-color:#d4dae6;
+}
+
+header ul {
+  display: flex;
+  background-color: #9dbbd5;
+  width: 100%;  
+}
+li {
+  flex: 1;
+  text-align: center;
+  padding: 2rem;
+  font-weight: bold;

```

p2: `font-weight: bold`보다는 정확한 수치를 나타내주는 것이 일관성 유지, 정확한 표현 측면에서 더 좋은거 같아요!
```suggestion
  font-weight: 700;
```
추가적으로, bold는 700으로 매핑되고, normal은 400으로 매핑된다고 하니 참고하세요!

---

오 처음알았어요..! 👍